source/backend/api/Areas/Reports/Controllers/AcquisitionController.cs

```diff
@@ -74,6 +79,49 @@ public IActionResult ExportLeases([FromBody] AcquisitionReportFilterModel filter
 
             return ReportHelper.GenerateExcel(reportAgreements, "Agreement Export");
         }
+
+        /// <summary>
+        /// Exports compensation requisitions as Excel file.
+        /// Include 'Accept' header to request the appropriate export -
+        ///     ["text/csv", "application/application/vnd.ms-excel"].
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
```

Comment:
nit: add a return description for summary.

Reply:
updated